modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -1,3 +1,63 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+Subscriptions are a core concept of GraphQL and one which allow us to enable real time updates and synchronisation of data.
```

Comment:
I will remove `synchronization of data`. Subscriptions are sadly one way only. 
There are concepts in community for two way subscriptions (using socket for queries/mutations) so it is best to make it clear that we support one way only.